Chap_API_Proc_Mgmt.tex

```diff
@@ -432,7 +448,7 @@ \subsection{Spawn attributes}
 }
 %
 \declareAttribute{PMIX_INDEX_ARGV}{"pmix.indxargv"}{bool}{
-Mark the \code{argv} with the rank of the process.
+If set to true, will use the given name of the executable (\code{argv[0]}) as a base name and each rank will be invoked using the base name with the string "-<\emph{rank}>" appended to it, where \emph{rank} is the \ac{PMIx} rank of the process being invoked.
```

Comment:
Should probably make clear that PMIx isn't doing this - we are passing an attribute to the host environment requesting that _it_ do this. Whether or not it happens is completely up to the host.

Reply:
Just noting that the change adds language like "...will use...", but leaves hanging the question of "who" will use the name. If you want to clarify, it might be something like "host is requested to use...". Or you can split this list to indicate which attributes are passed to the host for execution. Not entirely sure of the best way. Just trying to help you in your quest to clarify exactly what we mean here, and not replace one confusion with another.